elastic/kibana
# [Infra UI] Show syntax error for the Processes search input

Closes #164970 
Fixes https://github.com/elastic/kibana/issues/165287

## Summary

- Adds an error message to the Processes tab for cases when user types an invalid term into the search field.
- Also fixes the issue when the search field (on all tabs) would lose focus while typing

**Showing the error**
https://github.com/elastic/kibana/assets/793851/a56ba086-decc-49f8-8a51-5e44ddd17d1d

**Before the change, field losses focus**
https://github.com/elastic/kibana/assets/793851/23cb2435-ec5c-4e3a-b955-97f2eca03307

**After the change**
https://github.com/elastic/kibana/assets/793851/a2c23da0-0d5f-4f7e-8835-e1d2e4eed4e6


## How to test
- Checkout locally
- Open host details as a page
- Go to the Processes tab and type "," in to the search field
- Make sure the app does not crash and shows the error
- Check the same for host details in the flyout
